pkg/monitoring/clusterrolebinding_test.go

```diff
@@ -15,7 +15,7 @@ func TestGetSystemClusterRoleBindings(t *testing.T) {
 	clusterName := "test-cluster"
 	clusterRoleBindings := GetSystemClusterRoleBindings(clusterName)
 	assert.NotNil(clusterRoleBindings, "Expected the response to not be nil")
-	assert.Len(clusterRoleBindings, 3, "Expected three ClusterRoleBindings to be returned")
+	assert.Len(clusterRoleBindings, 4, "Expected three ClusterRoleBindings to be returned")
```

Comment:
change "three" to "four"? (Or change to use digit '4', so future changes won't miss the spelled number again.)

Reply:
fixed